timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -936,6 +940,88 @@ def evaluate(
             )
         return scores_dict
 
+    def get_feature_importance(
+        self,
+        data: TimeSeriesDataFrame,
+        features: Optional[List[str]] = None,
+        model: Optional[Union[str, AbstractTimeSeriesModel]] = None,
+        metric: Optional[Union[str, TimeSeriesScorer]] = None,
+        time_limit: Optional[float] = None,
+        method: Literal["naive", "permutation"] = "permutation",
+        subsample_size: int = 50,
+        num_iterations: int = 1,
+        random_seed: Optional[int] = None,
+    ) -> pd.DataFrame:
```

Comment:
Is there a reason that we aren't re-using Tabular's feature importance logic? https://github.com/autogluon/autogluon/blob/master/core/src/autogluon/core/utils/utils.py#L688-L929

Reply:
The data format... We have both "static" and "dynamic" features that need to be treated separately as well as those dynamic features that should only be shuffled in the past or those only in the future. As usual, the `X, y` API does not carry over in this case.